packages/domain-picker/src/domain-picker/suggestion-item.tsx

```diff
@@ -106,20 +106,15 @@ const DomainPickerSuggestionItem: React.FC< Props > = ( {
 			? __( 'Default', __i18n_text_domain__ )
 			: __( 'Free', __i18n_text_domain__ );
 
-	const firstYearIncludedInPaid = isMobile
-		? __( 'Included in paid plans', __i18n_text_domain__ )
+	const paidIncludedDomainLabel = isMobile
+		? __( 'Included in annual plans', __i18n_text_domain__ )
 		: createInterpolateElement(
-				__( '<strong>First year included</strong> in paid plans', __i18n_text_domain__ ),
+				__( '<strong>First year included</strong> in annual plans', __i18n_text_domain__ ),
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.